### DISCUSSION TITLE
Question: Ok So I Seen A Couple Of Mods Have Multiple Icons Like When Boyfriend Smiles On Ugh Hig Effort When All To The Left How Do I Do That

### DISCUSSION THREAD

**Discussion by ghost** [FunkinCrew/Funkin#1565] [Q&A]

**Comment by libauror-a** [comment]
1. Use the body of your question
2. Just use FPS Plus because thats what High Effort Ugh was made in

**Comment by bonkmaykr** [comment]
Please don't write the entire question in the Issue title. It's very hard to read.

The title should be a brief summary of the actual question or problem, usually 10 words or less.

**Reply by isakube** [reply]
github bot type response

**Comment by ghost** [comment]
> 1. Use the body of your question
> 2. Just use FPS Plus because thats what High Effort Ugh was made in

ProjectFNF might be better. Depends whether FPS Plus uses OG input.

**Comment by ghost** [comment]
no

**Reply by NokioPyro** [reply]
Best response to your own post
